tests/testthat/test-helpers_knitr_Stopping.R

```diff
@@ -108,6 +121,18 @@ test_that("knit_print.StoppingList works correctly", {
       "-  ≥ 20 patients dosed: If 20 or more participants have been treated.\n\n\n\n"
```

Comment:
You will also need to check that the HTML files in `/testthat/snaps/helpers_knitr` have updated correctly.  They are created (and checked)  at line 90 of `/testthat/test-helpers_knitr.R`.